### PR TITLE
feat(protocol-designer): replace 200µl tiprack with 300µl tiprack

### LINKED ISSUE #1955: PD Labware: Add P300 TipRack; Remove P200 TipRack
As a protocol designer, I would like to be able to use the P300 tiprack.

## Acceptance Criteria
- Add P300 tip rack to Protocol Designer
- Remove P200 tip rack from Protocol Designer

## Implementation Notes
- We currently only have the P200 tip rack

### PR DESCRIPTION
**Pull request by b-cooper**
## Overview

Remove the old 200µl tiprack as an option in labware and replace it with the 300µl tiprack.

Closes #1955

### CODE REVIEW

**Comment by IanLondon** [comment]
Also looks like this in deck setup mode and final deck state...
image

**Comment by IanLondon** [comment]
If you change `"format": "96-standard"` to `"format": "tiprack"` then it should work normally. Locally you'd need to run `make -C shared-data build` to see the update

**Comment by howisthisnamenottakenyet** [comment]
@b-cooper imported a protocol from production that uses a p200 tip rack. On deck setup I can't add more p200 tipracks. Is it possible to let that happen?

Ideally you wouldn't be able to create *new* protocols with a p200 tip rack but you would be able to keep place p200s in old protocols with p200 tip racks.

File: protocol-designer/src/labware-ingred/reducers/index.js [hunk 1]
```diff
@@ -325,8 +326,9 @@ const labwareOptions: Selector<Options> = createSelector(
   getLabware,
   getLabwareNames,
   (_labware, names) => reduce(_labware, (acc: Options, labware: Labware, labwareId): Options => {
-    // TODO Ian 2018-02-16 more robust way to filter out tipracks?
-    if (!labware.type || labware.type.startsWith('tiprack')) {
+    const labwareDef = getLabwareDef(labware[labwareId].type)
+    const isTiprack = labwareDef && labwareDef.metadata.isTiprack
```
Comment (IanLondon) [inline protocol-designer/src/labware-ingred/reducers/index.js]: same

File: protocol-designer/src/file-data/selectors/commands.js [hunk 2]
```diff
@@ -68,9 +69,10 @@ export const getInitialRobotState: BaseState => StepGeneration.RobotState = crea
     const tipracks: TiprackTipState = reduce(
       labware,
       (acc: TiprackTipState, labwareData: StepGeneration.LabwareData, labwareId: string) => {
-        // TODO Ian 2018-05-18 have a more robust way of designating labware types
-        // as tiprack or not
-        if (labwareData.type && labwareData.type.startsWith('tiprack')) {
+        const labwareDef = getLabware(labware[labwareId].type)
+        const isTiprack = labwareDef && labwareDef.metadata.isTiprack
```
Comment (IanLondon) [inline protocol-designer/src/file-data/selectors/commands.js]: this should use `getIsTiprack` getter fn from shared-data

File: protocol-designer/src/labware-ingred/reducers/index.js [hunk 3]
```diff
@@ -325,8 +326,8 @@ const labwareOptions: Selector<Options> = createSelector(
   getLabware,
   getLabwareNames,
   (_labware, names) => reduce(_labware, (acc: Options, labware: Labware, labwareId): Options => {
-    // TODO Ian 2018-02-16 more robust way to filter out tipracks?
-    if (!labware.type || labware.type.startsWith('tiprack')) {
+    const isTiprack = getIsTiprack(labware[labwareId].type)
```
Comment (IanLondon) [inline protocol-designer/src/labware-ingred/reducers/index.js]: PD won't load, white screens on load - I think the `.type` in this line 329 shouldn't be there

**Pull request merged**